opm/parser/eclipse/Deck/Deck.hpp

```diff
@@ -118,6 +120,8 @@ namespace Opm {
             void initUnitSystem() const;
 
             std::vector< DeckKeyword > keywordList;
+            std::shared_ptr<MessageContainer> m_messageContainer;
+
```

Comment:
Does it have to be a shared_ptr?  Couldn't it just be `MessageContainer m_messageContainer;` ?